articles/extensions/authorization-extension/v2/rules.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Using Rules with the Authorization Extension
+description: How to use information from the extension in rules
+---
+
+# Authorization Extension: Rules
+
+With [rules](/rules) that execute during the login process, you can grant them permissions to reach out to the Authorization Extension to do things like:
+
+1. Add custom claims to the issued token
```

Comment:
This looks more like a bulleted list, it's not a list of sequential stuff

Reply:
How does the first item differs from the last? (adding groups, roles, etc in token)